### DISCUSSION TITLE
Discord TOS

### DISCUSSION THREAD

**Discussion by cainthebest** [SanjaySunil/BetterDiscordPanel#39] [Q&A] [answered]
Does the better discord panel follow all TOS, privacy policy and etc due to the fact i would like it to be clear if it does. Thanks

**Comment by devilAPI** [comment]
Well as long as you login as a Bot and dont abuse your Powers, i dont think it breaks the ToS.

**Comment by SanjaySunil** [comment, marked as answer]
BetterDiscordPanel is safe to use and follows the Discord ToS. As mentioned already by devilAPI, abusing the functions of the panel and using a user account can get you banned.
